README.md

```diff
@@ -0,0 +1,26 @@
+# Field Formatter
+
+[Field Formatter](https://www.drupal.org/project/field_formatter) module adds formatters for entity reference fields to output only a specific field or field value using a specific formatter w/o needing to configure media view mode and configuring it to display properly. This module provides two formatters: *Field formatter with inline settings* and *Field formatter from view display*.
+
+## Installation
+
+1. Download [Field Formatter](https://www.drupal.org/project/field_formatter) from [GitHub](https://github.com/drupal-media/field_formatter).
+2. Install it in the [usual way](https://www.drupal.org/documentation/install/modules-themes/modules-8).
+
+## Usage
+
+1. Create entity reference field
```

Comment:
Let's use some punctuation here. Maybe a colon after main items and period (or whatever makes sense) after sub-items.